admin/views/form/select.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+if ( ! defined( 'WPSEO_VERSION' ) ) {
+	header( 'Status: 403 Forbidden' );
+	header( 'HTTP/1.1 403 Forbidden' );
+	exit();
+}
+
+/**
+ * @var string $select_class    The class name for the select.
+ * @var string $select_name     Value for the select name attribute.
+ * @var string $select_id       ID attribute for the select.
+ * @var array  $select_options  Array with the options to show.
+ * @var string $selected_option The current set options.
+ */
```

Comment:
Prefixing these variables has no added value as the filename already gives away that this is for a select element.